layouts/partials/home/contributors.html

```diff
@@ -8,7 +8,16 @@ <h2 class="border-b-2 border-green inline-block max-w-sm self-center">
         <p class="mt-12 mb-12">
           This site and the listings are all available because of contributors like you. If you’re looking to contribute to any of the listings or to this site, take a look at the contributing guidelines in the repository below.
```

Comment:
These changes look great but maybe we should modify the copy here since there aren't actually contributing guidelines being referenced below. I forgot to modify this when I changed the UI last time. In this case, we can just remove the second sentence?

Reply:
This is non-blocking though

Reply:
Oops. I might need that review again to merge since I changed it.